pkg/api/chunk.go

```diff
@@ -70,6 +70,8 @@ func (s *Service) chunkUploadHandler(w http.ResponseWriter, r *http.Request) {
 			jsonhttp.NotFound(w, "batch with id not found")
 		case errors.Is(err, errInvalidPostageBatch):
 			jsonhttp.BadRequest(w, "invalid batch id")
+		case errors.Is(err, ErrDevNodeNotSupported):
```

Comment:
Why are we handling this error here? We only get the error once the putter is used. We should not get this error here.

Reply:
Forgot to remove this.